File: Python/factorial.py
def fact(x):
    result = 1
    for i in range(1,x):
        result = result * (i+1)
    return result


if __name__ == "__main__":
    n = int(input("Enter the number: "))
    print("The factorial of {} is: {} ".format(n, fact(n)))

File: Dart/maths/Factorial.dart
import 'dart:io';

// Calculate factorial of number
int get_factorial(int number) {
  int count = 1;
  int factorial = 1;

  // Iterate from 1 till number
  while (count <= number) {
    factorial *= count;
    count++;
  }

  return factorial;
}

// Main Function, with driver code
void main() {
  print("Enter a number:");
  int input = int.parse(stdin.readLineSync()!);

  // Call factorial function
  int factorial = get_factorial(input);
  print("Factorial of $input is $factorial");
}
/**
 *  Sample input/output:
 * Enter a number:
 * 5
 * Factorial of 5 is 120
 * 
 * Enter a number:
 * 12
 * Factorial of 12 is 479001600
 */
